common/remote/sockfile.cpp

```diff
@@ -1796,6 +1796,18 @@ class CRemoteFilteredFileIOBase : public CRemoteBase, implements IRemoteFileIO
         // NB: inputGrouped == outputGrouped for now, but may want output to be ungrouped
 
         openRequest();
+        if (queryOutputCompressionDefault())
```

Comment:
Could the compression type ever be something like "NONE" in which case the query of it is not a nullptr but we do not want to create an expander ?

Reply:
I don't think that's necessary, but if it were set to anything unknown the block below would handle it and warn that unknown.
However, I think if the setRemoteOutputCompressionDefault may be called at the moment with "" if the property isn't configured, which would lead to a spurious warning that "" is an unknown compressor.
I'll fix that.